src/core/models/outputs/storage_deposit.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CORE_MODELS_OUTPUT_STORAGE_DEPOSIT_H__
+#define __CORE_MODELS_OUTPUT_STORAGE_DEPOSIT_H__
+
+#include "core/models/outputs/byte_cost_config.h"
+#include "core/models/outputs/outputs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Check if a sufficient storage deposit was made for the given output
+ *
+ * @param[in] config A byte cost configuration
+ * @param[in] output_type UTXO output type
+ * @param[in] output Pointer to an output
+ * @return true if output has enough storage deposit amount
+ */
+bool storage_deposit_sufficient_output_deposit_check(byte_cost_config_t *config, utxo_output_type_t output_type,
```

Comment:
we know it checks if the output has sufficient deposit, therefore `sufficient output_deposit` is redundant, the function name can be shorter.
```suggestion
bool storage_deposit_check(byte_cost_config_t *config, utxo_output_type_t output_type,
```

Reply:
Done. Thanks.